src/piv.rs

```diff
@@ -767,6 +767,7 @@ impl RsaKeyData {
     /// Generates a new RSA key data set from two randomly generated, secret, primes.
     ///
     /// Panics if `secret_p` or `secret_q` are invalid primes.
+    #[allow(clippy::unwrap_used)] // TODO(tarcieri): make fallible and handle errors
```

Comment:
Seems like there are a lot of situations where malformed/malchosen `p`/`q` will currently cause this code to crash